hummingbot/VERSION

```diff
@@ -1 +1 @@
-dev-1.29.0
+2.0.0
```

Comment:
@rapcmia we need to update the dev version right?

Reply:
Yes should be dev-2.0.1 for now
https://github.com/hummingbot/hummingbot/blob/development/hummingbot/VERSION